pkg/vault/test/foundry/unit/VaultUnitSwap.t.sol

```diff
@@ -43,9 +44,12 @@ contract VaultUnitSwapTest is BaseTest {
     uint256[] decimalScalingFactors = [uint256(1e18), 1e18];
     uint256[] tokenRates = [uint256(1e18), 2e18];
 
+    IProtocolFeeController feeController;
+
     function setUp() public virtual override {
         BaseTest.setUp();
         vault = IVaultMock(address(VaultMockDeployer.deploy()));
+        feeController = vault.getProtocolFeeController();
```

Comment:
Maybe we can put this in the base layer, and make `_getAggregateFeePercentage` use it?

Reply:
(or just keep the code as is now, but store `feeController` reference in the base test to reduce setup in every test)

Reply:
Moved it up to the BaseVaultTest